common/display/displayqueue.cpp

```diff
@@ -215,8 +223,19 @@ bool DisplayQueue::QueueUpdate(std::vector<HwcLayer*>& source_layers) {
   return true;
 }
 
+void DisplayQueue::GetNextQueueItem(DisplayQueueItem& item) {
+  display_queue_.lock();
```

Comment:
No need for the locks here. The callers are already holding the locks. Perhaps u want to separate this change to a separate commit and next commit can just use it. We can merge this cleanup in mean time. GetNextQueueItem being called from HandleUpdateRequest I mean.